server/schemas/mutations/editJob.js

```diff
@@ -0,0 +1,9 @@
+const Job = require('../../models');
+
+async function editJob(parent, { updateJob}) {
```

Comment:
Add in jobId import here along with updateJob, same for editProposal

Reply:
Bring in context here as well and use to complete user authorization before allowing edit. Similar to what you're doing in the delete resolvers